src/main/target/TINYFISH/target.h

```diff
@@ -0,0 +1,122 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define USB_VCP_ENABLED 1
+
+#define TARGET_BOARD_IDENTIFIER "tFSH" // http://fishpepper.de/projects/tinyFISH
```

Comment:
Might be better to go for `TFSH` here. All the other boards use capitals / numbers only, and the board identifier is something that is read by pretty much anything that 'speaks' MSP.